crates/compiler/gen_llvm/src/llvm/externs.rs

```diff
@@ -160,9 +160,12 @@ pub fn add_default_roc_externs(env: &Env<'_, '_, '_>) {
             }
         }
 
-        unreachable_function(env, "roc_getppid");
-        unreachable_function(env, "roc_mmap");
-        unreachable_function(env, "roc_shm_open");
+        #[cfg(not(windows))]
+        {
+            unreachable_function(env, "roc_getppid");
+            unreachable_function(env, "roc_mmap");
+            unreachable_function(env, "roc_shm_open");
+        }
```

Comment:
this will not work when cross-compiling. we need to check the target we generate code for, not the target we're compiling on.

Reply:
Updated to use `target_os = "windows"` which works well.